src/DGtal/topology/KhalimskyPreSpaceND.h

```diff
@@ -1355,6 +1355,23 @@ namespace DGtal
     */
     static SCell sIndirectIncident( SCell p, Dimension k );
 
+    
+    /** @brief For a given (n-1)-signed pre-cell, returns its interior voxel
```

Comment:
a (n-1)-signed pre cell  is a surfel.

In the usual  terminology (Herman, Udupa), a n-cell is a spel. I think I have used also this terminology in some methods (like uSpel, sSpel, etc).

However, since you wish here to return only the coordinates of the spel, I think "voxel" might be appropriate.